src/main/java/org/gitlab4j/api/GroupApi.java

```diff
@@ -224,6 +225,32 @@ public GroupApi(GitLabApi gitLabApi) {
         return (new Pager<Group>(this, Group.class, itemsPerPage, formData.asMap(), "groups", groupId, "subgroups"));
     }
 
+    /**
+     * Get a list of projects belonging to the specified group ID.
+     *
+     * GET /groups/:id/projects
+     *
+     * @param groupId the group ID to list the projects for
```

Comment:
The below ```@params``` need to be removed or an error occurs when the Javadocs are generated.

Reply:
Oh sorry, forgot to keep the doc in sync.